src/lib/mina/local-blockchain.ts

```diff
@@ -117,147 +119,152 @@ function LocalBlockchain({
     getNetworkState() {
       return networkState;
     },
-    async sendTransaction(txn: Transaction): Promise<PendingTransaction> {
-      let zkappCommandJson = ZkappCommand.toJSON(txn.transaction);
-      let commitments = transactionCommitments(
-        TypesBigint.ZkappCommand.fromJSON(zkappCommandJson),
-        minaNetworkId
-      );
+    sendTransaction(
+      txn: Transaction<boolean, boolean>
```

Comment:
Do we want to constrain `sendTransaction` to only proven and signed txs? We could provide means of overriding this for tests.

Reply:
I wouldn't, not every tx needs a proof